src/main/java/bio/terra/cli/command/app/passthrough/Gcloud.java

```diff
@@ -15,4 +37,49 @@ public String getExecutableName() {
   public String getInstallationUrl() {
     return "https://cloud.google.com/sdk/docs/install";
   }
+
+  @Override
+  protected void executeImpl() {
+    workspaceOption.overrideIfSpecified();
+
+    if (bucketName != null && builds && submit) {
+      var resource = Context.requireWorkspace().getResource(bucketName);
+      if (Resource.Type.GCS_BUCKET != resource.getResourceType()) {
+        throw new UserActionableException(
+            String.format(
+                "%s %s cannot be cloned because it is not a gcs-bucket",
```

Comment:
You're not cloning anything. So all `clone` should be removed.

When running `terra gcloud builds submit`, you must pass the resource name of a GCS bucket with `--gcs-bucket`.